app/static/src/scss/base.scss

```diff
@@ -0,0 +1,13 @@
+.banner__container {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: none;
+}
+
+.sign-out {
+    font-size: 1.25rem;
```

Comment:
should we be using rem or gds font sizes? https://design-system.service.gov.uk/styles/typography/#:~:text=your%20own%20components.-,Font%20size,to%2080px%20on%20large%20screens.
https://designnotes.blog.gov.uk/2018/02/19/developing-new-typography-and-spacing-for-gov-uk-frontend/